test/factories/medium.rb

```diff
@@ -18,6 +18,18 @@
       os_family { 'Coreos' }
     end
 
+    trait :fcos do
+      sequence(:name) { |n| "Fedora CoreOS Mirror #{n}" }
+      sequence(:path) { 'http://builds.coreos.fedoraproject.org' }
```

Comment:
should be also https

Reply:
this is irrelevant as it is only for test, the url is never actually contacted.